xc7/utils/prjxray_form_channels.py

```diff
@@ -82,28 +205,42 @@ def import_tile_type(db, write_cur, tile_types, site_types, tile_type_name):
     tile_types[tile_type_name] = write_cur.lastrowid
 
     wires = {}
-    for wire in tile_type.get_wires():
+    for wire, wire_rc_element in tile_type.get_wires().items():
+        capacitance = 0.0
```

Comment:
Shouldn't we insert here some reasonable dummy variables other than zero ? Zeros might trigger some assertions in the VPR or lead to some math exceptions / corner cases.

Reply:
0 is not only legal, but most wires in the actual timing model use zeros.